server/include/data_store.hpp

```diff
@@ -40,8 +41,11 @@ class DataStore
 
     bool UpdateDeviceLocation(const std::string& device_id, Position pos);
 
-    bool
-    InsertRSSIReadings(const std::string& device_id, std::vector<std::string> mac_addr_list, std::vector<double> rssi_list);
+    bool AssignDeviceToEmployee(const std::string& device_id, const std::string& employee_id);
+
+    bool InsertRSSIReadings(const std::string&       device_id,
+                            std::vector<std::string> mac_addr_list,
+                            std::vector<double>      rssi_list);
```

Comment:
From what I understand, we are passing a vector of mac addresses and another vector of rssi. How do we make sure they are matching or what is the relationship between those two?

Reply:
These [ lines of code](https://github.com/platisd/indoor-navigation-system/blob/dev/server/src/data_store.cpp#L102-L106) ensures that we have equal size, before we proceed with the insert.
It is 1-1 mapping.

Reply:
Since it is a public function, we could in theory not be sure how it is used. What about using a vector of [std::pair](http://www.cplusplus.com/reference/utility/pair/)?

Reply:
Done.